src/options.rs

```diff
@@ -172,7 +173,7 @@ pub struct IgnoreList {
 pub fn build_app(
     matches: &ArgMatches, config: &mut Config, widget_layout: &BottomLayout,
     default_widget_id: u64, default_widget_type_option: &Option<BottomWidgetType>,
-    colours: &CanvasColours,
+    expanded_upon_startup: bool, colours: &CanvasColours,
```

Comment:
nit: Could also just grab `expanded_upon_startup` within this function instead, I guess, as opposed to passing it around.